_posts/2017-11-25-book-review-introduction-to-service-design.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Book Review: How To Have Your Cake And Eat It Too: An Introduction to Service Design"
+published: 2017-11-25 23-27-41 +00:00
+tags: book service-design
+---
+
+I am a software developer, but also a business consultant. Every year I work
+with a bunch of small companies, founders and those who are running teams in
+much larger organisations. These are people who /do/care about the quality of
+the code I might produce, how well tested it is and that decisions have been
+made for the long term, but it’s all a bit moot if we’re not building the right
+thing for their customers.
+
+Over the last few years, government and large enterprises have gradually
+started to come around to what’s made new upstarts so successful: focusing on
+the user and their needs.
+
+For the new upstarts, some of this has been easy. They don’t have decades worth
+of processes and organisational cruft which stands in the way of delivering on
+their goals. For successful new companies, they can be arranged from the
+customers’ perspectives from the start. If they’re not, they’re young and small
+enough to iterate towards this quickly. For existing organisations, this is
+rarely the case.
+
+Service Design is a collection of tools which allow you to find out what
+customers actually value in a given service or product.
+
+* Who are your customers?
+* What journey does a customer take in using your service?
+* What are they trying to achieve?
+* What are their pain points?
+
+These can be answered by building personas, drawing out a customer journey map
+and then doing customer interviews to fill in the gaps, respectively.
+
+The consultant is merely a product of the tools in their toolbox. I’ve
+participated in several, and since run, a design sprint. I’ve built products
+using various flavours (and in various levels of dysfunction) of agile. But I’d
+been hunting for something out of those which is missing, and a step into this
+“service design” thing I’d been hearing about regularly.
+
+J. Margus Klaar presents us with brief overview of all of the above. At 106
+   pages, it’s short enough to read and digest in a couple of hours and present
+   you with another collection of tools to have at your disposal.
+
+They discuss how consumer actions have changed over the past fifty years and
+changes of the world which are causing peoples’ behaviour to change. They
```

Comment:
"Change of the word" doesn't quite make sense to me, did you mean "to the world"?